app/abilities/organization.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+import { Ability } from 'ember-can';
+
+export default Ability.extend({
+  isAtLeastAdmin: Ember.computed.or('membership.isAdmin', 'membership.isOwner'),
+  userCanJoinOrganization: Ember.computed.empty('membership'),
+  userCanLeaveOrganization: Ember.computed.or('membership.isContributor', 'membership.isAdmin'),
+  userIsMemberInOrganization: Ember.computed.notEmpty('membership'),
+
+  canJoin: Ember.computed('membership', 'membership.role', 'model', function() {
+    return this.get('userCanJoinOrganization');
+  }),
```

Comment:
This should probably just be an alias, as in `canJoin: Ember.computed.alias('userCanJoinOrganization')`. If I'm understanding my ember correctly, everything should be have the same.